internal/runtime/runtime_test.go

```diff
@@ -76,7 +76,8 @@ func TestProcessMessageBusRequest(t *testing.T) {
 
 	runtime := NewGolangRuntime("", nil, dic)
 	runtime.SetDefaultFunctionsPipeline([]interfaces.AppFunction{dummyTransform})
-	result := runtime.ProcessMessage(context, envelope, runtime.GetDefaultPipeline())
+
+	_, result, _ := runtime.DecodeMessage(context, envelope)
```

Comment:
```suggestion
	_, decodeErr, _ := runtime.DecodeMessage(context, envelope)
```

you use `result`, `err`, and `decodeErr` inconsistently throughout this file for the same value

Reply:
agreed. I think that comes from originally the return value was an interface{} which could be data or an error. 
Fixed in this instance.